main.hpp

```diff
@@ -49,18 +52,23 @@ class GameData {
         this->functions[SERWEROWNIA] = &fail;
         this->functions[WINDA_PIETRO] = &windaPietro;
         this->functions[WINDA_PARTER] = &windaParter;
-        this->functions[WINDY_TYLNE] = &fail;
+        this->functions[WINDY_TYLNE] = &windyTylne;
+        this->functions[BIBLIOTEKA] = &fail;
         this->functions[GARAZ] = &fail;
         this->functions[ZEWNATRZ] = &fail;
         this->functions[FAIL] = &fail;
     }
 };
 
 bool yes();
+bool yes_arg (std::string s);
```

Comment:
Ta funkcja też mogłaby się nazywać yes()

Reply:
Nie powiedziałbym, gdyż różni się od poprzedniej. Pierwsza porównuje do "YyTt", a moja do podanych argumentów, więc powinny się nazwą różnić

Reply:
Hm, tak naprawdę bezargumentowa yes() to szczególny przypadek tej, w sumie można by ją zaimplementować jako `return yes_arg("yYtT");`

Reply:
Wgl tak superpoprawnie to ja bym sugerował zmienić nazwę `yes_arg` na `yes` z taką sygnaturą

`bool yes(const std::string& s)`

chyba, że nie mówił @phoe o constach i referencjach, to się nie oddzywałem.

Reply:
Nie mówiłem jeszcze o constach i referencjach, ale dobrze, że wspominasz - na kolejnym ksicampie się o tym powie.